<?xml version = "1.0"?>

<launch>
    <!-- Robot Localization Node -->
    <node name="ukf_localization_node"
        pkg="robot_localization"
        type="ukf_localization_node"
        clear_params="true">
        <rosparam file="$(find rover)/params/localization.yaml"/>
        <param name ="/use_sim_time" value="true"/>
    </node>

    <!-- <node name="localizationNode" pkg="rover" type="localization" respawn="false" output="log"/> -->

    <!-- <rosparam command="load" file="$(find rover)/params/localization.yaml" />

    <node pkg="robot_localization" type="ekf_localization_node" name="ekf_se_odom" clear_params="true"/>

    <node pkg="robot_localization" type="ekf_localization_node" name="ekf_se_map" clear_params="true">
        <remap from="odometry/filtered" to="odometry/filtered"/>
    </node> -->
    <!-- <node name="poseDPublisher_node" pkg="rover" type="posePublisher" respawn="false" output="screen"/> -->
</launch>